### DISCUSSION TITLE
Huge Shout Out to Zerfl

### DISCUSSION THREAD

**Discussion by LukeCroteau** [LukeCroteau/RaidExtractor#61] [General]
I just wanted to make a point to throw a HUGE Shoutout to @zerfl 

They have definitely been contributing a TON while I'm still adjusting to my new job, with updates ranging from the Hero Types file, to some cool additions like the Mercy information and Battle Presets!

Thanks so much for helping contribute!

**Comment by zerfl** [comment]
Aww, thank you for your encouraging words. That's very kind of you. It's been a lot of fun working on this project and it definitely helped rekindle my love for programming and some RE, albeit on a "hobby"-level.

One thing to note, though. I can't take credit for the introduction of shard extractions / mercy system as this was contributed by the wonderful @SteveJRose.

Thanks again!

**Reply by LukeCroteau** [reply]
Oh my god, you're right! I'm so sorry!

I think what I had in my brain was the Great Hall and Arena information, since that was a very longstanding feature!
